rasa/nlu/training_data/formats/rasa_yaml.py

```diff
@@ -142,7 +170,7 @@ def _parse_training_examples(
         if isinstance(examples, list):
             example_strings = [
                 # pytype: disable=attribute-error
-                example.get(KEY_INTENT_TEXT, "")
+                example.get(KEY_INTENT_TEXT, "").strip("\n\r ")
```

Comment:
why do we need the strip?

Reply:
In testing phase I faced several problems with leading/trailing spaces and `\n` in the training examples, and found this as a safe solution to always take only "text" from the training examples/name of the intents.
At least I haven't seen the trailing white space as a vital part of a training data.

Reply:
makes sense :+1: